source/beamline/SXRMB/SXRMBBeamline.cpp

```diff
@@ -642,19 +642,22 @@ void SXRMBBeamline::setupSampleStage()
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageZ_);
 	solidStateSampleStageControlSet_->addControl(solidStateSampleStageR_);
 
-	// Ambiant Endstation sample state
+	// Ambiant Endstation sample stage
+	ambiantTableHeight_ = new AMPVwStatusControl("Ambiant Table Height", "TBL0000-E07-01:Y:mm:fbk", "TBL0000-E07-01:Y:mm", "TBL0000-E07-01:Y:mm:status", "BL1606-B1-1:AddOns:Ambiant:TableHeight:stop", this, 0.005, 2.0, new CLSMAXvControlStatusChecker());
```

Comment:
Which is a lazy solution?  Sorry I missed your last message while I was writing out my last one.